lib/membrane/video_compositor/video_compositor.ex

```diff
@@ -293,6 +288,17 @@ defmodule Membrane.VideoCompositor do
 
   @impl true
   def handle_setup(_ctx, opt) do
+    env = %{
```

Comment:
I don't like the fact that some options are only applied if compositor is starting the server. They will be silentlly ignored when compositor was already started. 

Note that this is not a new issue, current implementation has the same problem.